zk.js/tests/system-programs/functional_tests.ts

```diff
@@ -316,61 +334,77 @@ describe("verifier_program", () => {
       lamports: 1000 * 1e9,
       recipientPublicKey: origin.publicKey,
     });
+    const ata = await getAssociatedTokenAddress(MINT, origin.publicKey);
 
-    const txParams = new TransactionParameters({
-      inputUtxos: [
-        user.balance.tokenBalances
-          .get(tokenProgram.toBase58())!
-          .utxos.values()
-          .next().value,
-      ],
-      outputUtxos,
+    const unshieldUtxo = user.balance.tokenBalances
+      .get(tokenProgram.toBase58())!
+      .utxos.values()
+      .next().value;
+
+    // Running into memory issues with verifier one (10in2out) unshielding spl
```

Comment:
is this something to do about?

Reply:
This is not a new I issue, we just forgot about it. We will optimize memory use anyway so that will probably solve this issue as well. Another point is that we will probably remove shield and unshield functionality from verifiers by default that will also remove this issue.